src/virtualenv/create/creator.py

```diff
@@ -160,7 +169,8 @@ def run(self):
             safe_delete(self.dest)
         self.create()
         self.set_pyenv_cfg()
-        self.setup_ignore_vcs()
```

Comment:
As you can see from the name of this function the feature is nowhere bound to git. Is bound to VCS in general. Please rename all code/flag naming accordingly.

Reply:
Except that the feature is only implemented for Git ignore functionality, and there is no possibility of implementation for other VCS in the foreseeable future, due to the designs of those systems?

Can certainly change to `--no-vcsignore` but wondering if maintaining that agnosticism towards VCS is justified.

Reply:
We cannot think of just today and the system we support just today. When designing API for a widely used tool, we must be future proof. So yes, it is justified.

> there is no possibility of implementation for other VCS in the foreseeable future, due to the designs of those systems

I don't think this to be the case.